src/Sylius/Bundle/AddressingBundle/DependencyInjection/SyliusAddressingExtension.php

```diff
@@ -29,37 +29,19 @@ class SyliusAddressingExtension extends AbstractResourceExtension
      */
     public function load(array $config, ContainerBuilder $container)
     {
-        $config = $this->configure(
-            $config,
-            new Configuration(),
-            $container,
-            self::CONFIGURE_LOADER | self::CONFIGURE_DATABASE | self::CONFIGURE_PARAMETERS | self::CONFIGURE_VALIDATORS | self::CONFIGURE_FORMS | self::CONFIGURE_TRANSLATIONS
-        );
-
-        $container->setParameter('sylius.scope.zone', $config['scopes']);
+        $config = $this->processConfiguration(new Configuration(), $config);
+        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
```

Comment:
Don't you want to use the `DelegatingLaoder` which has been introduced since few weeks?